headers/types/dungeon_mode/dungeon_mode.h

```diff
@@ -177,14 +177,24 @@ struct statuses {
     bool power_ears;           // 0x50: STATUS_POWER_EARS
     bool scanning;             // 0x51: STATUS_SCANNING
     bool stair_spotter;        // 0x52: STATUS_STAIR_SPOTTER
-    undefined field_0x53;
+    // 0x53: Set when initally spawning a team member with the ability Pickup.
+    bool pickup_flag;
     bool grudge;       // 0x54: STATUS_GRUDGE
     bool exposed;      // 0x55: STATUS_EXPOSED (Foresight/Odor Sleuth)
     bool type_changed; // 0x56: Flag for if the monster's type has been changed
     bool boss_flag;    // 0x57: Seems to be true for boss monsters
-    undefined field_0x58;
+    // 0x58: Appears to be a flag for when a monster increasces their speed. Maybe only used
+    // by the RunLeaderTurn function to know if the leader has changed their speed stage partway
+    // through the function?
+    undefined unk_speed_boost_tracker;
+    // 0x59: Maybe related to being a team member and new recruit? Set to 1 in TryRecruit
+    // and 0 in SpawnTeam. Also checked in EnemyEvolution to be 0 before evolving. Maybe to
+    // prevent a recently recruited ally from evolving after and or to add a monster to the
+    // assembly after the completion of a dungeon?
     undefined field_0x59;
-    bool in_action;            // 0x5A: Possibly a flag while in action
+    // 0x5A: Possibly a flag while in action. Could also be a flag that to cause the  burn from
```

Comment:
There's a double-space before "burn"

Reply:
The description isn't very clear either. "A flag that to cause ..."? Or it that a typo?